datadog-trace-stats/src/span_concentrator/mod.rs

```diff
@@ -24,26 +25,25 @@ fn align_timestamp(t: u64, bucket_size: u64) -> u64 {
     t - (t % bucket_size)
 }
 
-/// Return true if the span has a span.kind that is eligible for stats computation
-fn compute_stats_for_span_kind<T>(span: &Span<T>, span_kinds_stats_computed: &[String]) -> bool
+/// Return true if the span is eligible for stats computation
+fn span_is_eligible<T>(span: &Span<T>, span_kinds_stats_computed: &[String]) -> bool
```

Comment:
nit: I prefer `is_span_eligible` (a question) over `span_is_eligible` (a statement). functions that return a bool should be named like a question the return value will answer.